helpers/api/proposal/read-refused.ts

```diff
@@ -51,16 +47,7 @@ export default async function readProposalRefused(events, network: Network_v2, c
           });
 
           if (proposal) {
-            if (network.contractAddress === publicRuntimeConfig?.contract?.address) {
```

Comment:
?

Reply:
@moshmage twitter integration will be made by webnetwork-events project